llvm/lib/Target/SPIRV/SPIRVInstructionSelector.cpp

```diff
@@ -34,6 +34,7 @@
 #include "llvm/IR/IntrinsicsSPIRV.h"
 #include "llvm/Support/Debug.h"
 #include "llvm/Support/ErrorHandling.h"
+#include "llvm/Support/raw_ostream.h"
```

Comment:
Not sure why you need an include your changes to  `selectExtInst` are the same string stream as the default intrinsic error reporting case: https://github.com/llvm/llvm-project/blob/6518b121f037717fd211c36659f7b25266424719/llvm/lib/Target/SPIRV/SPIRVInstructionSelector.cpp#L3124C5-L3128C48

Reply:
I don't recall adding that. I think my IDE did that for me :P 
I will remove it